data-showcase/src/main/user-interface/src/app/item-summary/item-summary.component.ts

```diff
@@ -22,7 +22,7 @@ export class ItemSummaryComponent implements OnInit {
   keywordsForConcept: string[] = [];
   environment: Environment;
 
-  constructor(private dataService: DataService,
+  constructor(public dataService: DataService,
```

Comment:
To make the fields inside services available to the html template, it is better to add a property to the component that reads from the service.